vdom/core.py

```diff
@@ -176,9 +194,15 @@ def from_dict(cls, value):
             validate(instance=value, schema=VDOM_SCHEMA, cls=Draft4Validator)
         except ValidationError as e:
             raise ValidationError(_validate_err_template.format(VDOM_SCHEMA, e))
+        attributes = value.get('attributes', {})
+        if 'style' in attributes:
+            style = attributes.pop('style')
```

Comment:
Why pop? Do you want to remove the attribute from the dict?

Reply:
I do! Since style is a dict, while all other attributes are strings.

Reply:
I meant you're removing elements from an argument dict that a caller may not expect to be mutated. Usually you want to avoid this side-effect for a function and either copy the dict first or make a new dict without the key when removing.